.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ version: 2
 jobs:
   py38:
     docker:
-      - image: cimg/python:3.8.12
+      - image: cimg/python:3.8.16
```

Comment:
There was a breaking change in pypi which made it incompatible with the version of Poetry pre-installed in the previous image versions